SPY-11630 All format in one project

fernandpeng opened the pull request:
Merged all format into one

ObjC & Swift both have a separated Workspace with different integration methods.

Commit:
All-format-in-one for swift direct integration

Pernic commented on Swift/Direct integration/Direct integration/Banner & MPU/BannerViewController.swift:
```diff
@@ -1,85 +1,82 @@
+//  Direct Integration Sample
 //
-//  Copyright © 2020 Ogury Co. All rights reserved.
+//  Created by Fernand Peng on 09/11/2021.
```
remark (blocking) : remove all 
`Created by Fernand Peng on 09/11/2021.`
on the top of file, you can write : 
` Copyright © 2020 Ogury Co. All rights reserved.`

fernandpeng replied:
Updated in 750d11cae96c04f3272b4a34935c4ea6ed56a689

Pernic commented on Swift/Direct integration/Direct integration/Banner & MPU/BannerViewController.swift:
```diff
@@ -1,85 +1,82 @@
 //
-//  ViewController.swift
-//  ThumbnailExample
+//  BannerManager.swift
```
remark(no-blocking): wrong name of file

fernandpeng replied:
Updated in 750d11cae96c04f3272b4a34935c4ea6ed56a689

Pernic commented on Swift/Direct integration/Direct integration/Banner & MPU/BannerViewController.swift:
```diff
@@ -1,85 +1,82 @@
+        self.addNewStatus("[Choice Manager] Loading...")
         
         //The setup of Ogury Choice Manager and Ogury Ads is done AppDelegate.swift file.
         OguryChoiceManager.shared().ask(with: self) { (error, answer) in
```
suggestion (no-blocking) : put Choice Manager on a different file

fernandpeng replied:
I think it's better to keep it in every ViewController so all of them have the whole logic complete example. (beside setup is done in AppDelegate and Constant store in separated file)

Pernic commented on Swift/Direct integration/Direct integration/Banner & MPU/BannerViewController.swift:
```diff
@@ -1,85 +1,82 @@
-        self.addNewStatus("MPU Loading ...")
-        mpuBanner = OguryAdsBanner.init(adUnitID: "ogury_adunit")
+        addNewStatus("[Ad][MPU] Loading ...")
+        mpuBanner = OguryAdsBanner.init(adUnitID: ConstantKeys.mpuAdUnit)
```
```suggestion
        mpuBanner = OguryAdsBanner(adUnitID: ConstantKeys.mpuAdUnit)
```

Pernic replied:
.init is not needed in Swift

fernandpeng replied:
Updated in 750d11cae96c04f3272b4a34935c4ea6ed56a689

Pernic reviewed (changes requested):
I don't have finish to read all the PR, but it's it's pretty good. I have make just small remark 👍

Commit:
Updated copyright on top of files, removed unnecessary .init, simplified UI part on storyboard and HomeViewController.

Pernic commented on .gitignore:
```diff
@@ -0,0 +1,5 @@
+*/Pods/
```
remark(no-blocking) : you can use some tools to generate the gitignore : https://www.toptal.com/developers/gitignore

fernandpeng replied:
I used this website as reference actually, but as the projet organised differently(there is ObjC and Swift as parent folder + one share workspace for Admob and direct integration) needed to customise a little bit :).

I will do the update for the ObjC version & Readme before merge this PR,
thanks in advance :)

Commit:
Update to latest version of OgurySdk, minor code & readme improvements

Pernic reviewed (commented):
The UI is not optimized for small iPhone.

Commit:
UI Fix

Commit:
UI adapt for small device

fernandpeng commented:
> The UI is not optimized for small iPhone.

fixed in 75a4bd7 - tested on iPod touch - 7th generation

Pernic commented:
MPU have not the good size :

https://user-images.githubusercontent.com/33022315/182330305-ef68047b-38b0-448f-81a5-d2120a68252d.mov

Commit:
Update constraint in banner view, added few comment

Commit:
Update admob integration

    pod 'OguryMediationGoogleMobileAds', '~> 2.1.0.0'
    pod 'Google-Mobile-Ads-SDK', '~> 9'

fernandpeng commented:
> MPU have not the good size :
> 
>  Screen.Recording.2022-08-02.at.10.32.46.mov

Fixed in d5280dd

fernandpeng commented:
@Pernic Applied feedbacks in d5280dd
and also updated admob integration in e722cd8

Commit:
SPY-11630 Fix UI error
